packages/xrpl/src/models/methods/federatorInfo.ts

```diff
@@ -0,0 +1,78 @@
+import { BaseRequest, BaseResponse } from './baseMethod'
+
+/**
+ * The `federator_info` command asks the federator for information
+ * about the door account and other bridge-related information. This
+ * method only exists on sidechain federators. Expects a response in
+ * the form of a {@link FederatorInfoResponse}.
+ *
+ * @category Requests
+ */
+export interface FederatorInfoRequest extends BaseRequest {
+  command: 'federator_info'
+}
+
+/**
+ * Response expected from a {@link FederatorInfoRequest}.
+ *
+ * @category Responses
+ */
+export interface FederatorInfoResponse extends BaseResponse {
```

Comment:
Should these values have docstrings/comments explaining what they are used for like we do with other requests?

Reply:
Yes - however, those don't exist yet.